saylua/modules/messages/messages.py

```diff
@@ -24,29 +45,19 @@ def messages_main_post():
     user_message_ids = request.form.getlist('user_conversation_id')
```

Comment:
Relying on this being untampered with seems unwise, but I could be misinterpreting this.